### PR TITLE
Add Hepatitis Data Set

### PR DESCRIPTION
**Pull request by kazu-5**
This pull request implements Support Hepatitis #36

reference
https://archive.ics.uci.edu/ml/datasets/hepatitis

### CODE REVIEW

File: lib/datasets/hepatitis.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
+require "csv"
+
+require_relative "dataset"
+
+module Datasets
+  class Hepatitis < Dataset
+    Record = Struct.new(
+      :Class,
```
Comment (kou) [inline lib/datasets/hepatitis.rb]: Could you use lower case for field name?
Because Ruby uses lower case for name normally.

Comment (kazu-5) [reply lib/datasets/hepatitis.rb]: thank you.
I will fix pointed matter.

**Comment by kou** [comment]
Thanks!

**Comment by kazu-5** [comment]
Thank you.

**Pull request merged**